neoload-writer/src/main/java/com/neotys/neoload/model/v3/writers/neoload/userpath/ThinkTimeWriter.java

```diff
@@ -1,30 +1,15 @@
 package com.neotys.neoload.model.v3.writers.neoload.userpath;
 
 import com.neotys.neoload.model.v3.project.userpath.ThinkTime;
-import com.neotys.neoload.model.v3.writers.neoload.ElementWriter;
-import org.w3c.dom.Document;
-import org.w3c.dom.Element;
 
-public class ThinkTimeWriter extends ElementWriter {
-
-    public static final String XML_TAG_NAME = "delay-action";
-    public static final String XML_DURATION_ATT = "duration";
-    public static final String XML_ISTHINKTIME_ATT = "isThinkTime";
+public class ThinkTimeWriter extends AbstractDelayActionWriter {
```

Comment:
This class is not part of the commit and is missing...

Reply:
Ah bad cut it is on other pr. So I will merge.